e2e-tests/commands/shared-commands/form/change-answer-radio-field.js

```diff
@@ -0,0 +1,15 @@
+import { radios, submitButton } from '../../../pages/shared';
+/**
+ * changes radio input/option
+ * @param {Object} fieldVariables
+ * @param {Function} input cypress input selector
```

Comment:
only 1 param being passed.  SHould this be returns?

Reply:
Typo, removed